src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlInstanceKeyVaultKey.md

```diff
@@ -117,25 +117,8 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -KeyId
-AzureKeyVault key id
-
-```yaml
-Type: String
-Parameter Sets: (All)
-Aliases:
-
-Required: True
-Position: 1
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
-### -ManagedInstance
-The managed instance input object
-
-```yaml
+### -Instance
+The instance input object```yaml
```

Comment:
Please move yaml tag to a new line.  Apply everywhere. If you upgrade to the latest version of platyPS you should not have this issue.